sound/soc/sof/Kconfig

```diff
@@ -28,7 +28,7 @@ config SND_SOC_SOF_ACPI
 	select IOSF_MBI if X86 && PCI
 	help
 	  This adds support for ACPI enumeration. This option is required
-	  to enable Intel Haswell/Broadwell/Baytrail/Cherrytrail devices
```

Comment:
@plbossart I have an ThinkPad X1 Carbon (about 5 years old) which has a HSW CPU 
https://ark.intel.com/content/www/us/en/ark/products/76308/intel-core-i5-4300u-processor-3m-cache-up-to-2-90-ghz.html

Reply:
I meant there are no Haswell devices with an I2S interface enabled, HDaudio is used in all commercial devices - and that prevents the use of the DSP.